content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -204,11 +204,7 @@ Because your Container did not specify its own CPU request and limit, it was giv
 [default CPU request and limit](/docs/tasks/administer-cluster/cpu-default-namespace/)
 from the LimitRange.
 
-At this point, your Container might be running or it might not be running. Recall that a prerequisite
-for this task is that your Nodes have at least 1 CPU. If each of your Nodes has only
-1 CPU, then there might not be enough allocatable CPU on any Node to accommodate a request
-of 800 millicpu. If you happen to be using Nodes with 2 CPU, then you probably have
-enough CPU to accommodate the 800 millicpu request.
+At this point, your Container might be running or it might not be running. Recall that a prerequisite for this task is that your cluster must have at least 1 CPU available for use. If each of your Nodes has only 1 CPU, then there might not be enough allocatable CPU on any Node to accommodate a request of 800 millicpu. If you happen to be using Nodes with 2 CPU, then you probably have enough CPU to accommodate the 800 millicpu request.
```

Comment:
Hi @davivcgarcia .
I am trying to understand the changes.
The original text states that:
`Each _node_ in your cluster must have at least 1 CPU.`
Is this statement no longer true?

This statement leaves room for interpretation:
`Your cluster must have at least 1 CPU available _for use_ to run the task examples.`

`Recall that a prerequisite for this task is that your cluster must have at least 1 CPU available for use.`
Again, I am trying to figure out what is implied by this statement.

If each of your [Nodes] nodes has only 1 CPU, then there might not be enough allocatable CPU on any Node to accommodate a request of 800 millicpu.

`If you happen to be using Nodes with 2 CPU`

Reply:
Hi @kbhawkey! That's exactly the point I discussed with @sftim in the issue #15123. My conclusion was that the original statement was referring to the minimal resource requirement for running the task's examples. So, in my opinion, the correct statement should be referring to cluster capacity not node sizing.

Reply:
Hi @davivcgarcia .
I think I understand now.
Do you think this paragraph should be a `note`?

Reply:
Good idea! I'll change that.